package.json

```diff
@@ -95,6 +95,7 @@
     "react": "16.13.1",
     "react-native": "^0.63.4",
     "react-native-macos": "^0.63.33",
+    "react-native-test-app": "^0.6.10",
```

Comment:
Why was this added? It shouldn't be necessary. Did you hit issues without it?

Reply:
No, I didn't hit any issues without it and it was auto-generated for me. So I thought I'd leave it that way. I will remove it.